giskard/datasets/base/__init__.py

```diff
@@ -507,6 +507,19 @@ def upload(self, client: GiskardClient, project_key: str):
             )
         return dataset_id
 
+    def extract_languages(self):
+        """
+        Extracts all languages present in the dataset 'text' column.
+
+        Returns:
+            list[str]: a list of language codescontaining all languages in the dataset.
```

Comment:
typo. What is language code? There are several standards. In our dataset meta, it's the [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1), maybe we can clarify it here.